polynote-frontend/polynote/ui.js

```diff
@@ -579,10 +613,19 @@ export class NotebookConfigUI extends UIEventTarget {
             }
         });
 
+        const sparkConfig = {};
+        const sparkConfigRows = this.sparkConfigContainer.querySelectorAll('.spark-config-row');
+        sparkConfigRows.forEach(row => {
+            const k = row.querySelector('.spark-config-key').value.trim();
+            const v = row.querySelector('.spark-config-val').value.trim();
+            if (k && v) sparkConfig[k] = v;
```

Comment:
@jeremyrsmith Do you know of any spark settings where intentionally setting the value to `""` is necessary?

Reply:
Hm, maybe just to be safe I should remove the check on `v` so people can do that if they want to...

Reply:
ok, removed the check on v, let people set blank values. Maybe it would be useful as a way to unset a config set by the environment or something.